src/freedesktop.rs

```diff
@@ -642,6 +639,13 @@ struct MountPoint {
     _mnt_fsname: String,
 }
 
+fn get_sorted_mount_points() -> Result<Vec<MountPoint>, Error> {
+    // Returns longest mount points first
+    let mut mount_points = get_mount_points()?;
+    mount_points.sort_by(|a, b| b.mnt_dir.as_os_str().as_bytes().len().cmp(&a.mnt_dir.as_os_str().as_bytes().len()));
```

Comment:
How about using `sort_unstable_by` instead of `sort_by` here?

Reply:
Haven't heard of that before, thanks for letting me know! I'll change this when I get back to my computer.